h/static/scripts/streamer.js

```diff
@@ -27,33 +20,18 @@ var socket;
  * @return The push notification service client.
  */
 // @ngInject
-function connect($rootScope, annotationMapper, groups, session, settings) {
-  // The public interface of the streamer. Returned from connect.
-  var controls = {
-    setConfig: setConfig
-  };
+function Streamer($rootScope, annotationMapper, groups, session, settings) {
+  // the randomly generated session UUID
```

Comment:
Nit. I know these comments were just moved down from above, but they should start with a capital letter.